www/src/css/_layout.scss

```diff
@@ -0,0 +1,92 @@
+@import './_var';
+
+.container {
+  margin: auto;
+  max-width: 1400px;
+}
+
+// Main
+// Two sections
+// main.njk
+.main {
+  display: flex;
+  .grid-toc {
+    position: fixed;
+    top: 44px;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    width: 100vw;
+    padding: 1rem;
+    flex-shrink: 0;
+    flex-grow: 0;
+    overflow: auto;
+    z-index: 100;
+
+    @media (min-width: $breakpoint-m) {
+      position: sticky;
+      top: 52px;
+      width: 320px;
+      max-height: calc(100vh - 52px);
+      overflow-y: scroll;
+      -webkit-overflow-scrolling: touch;
```

Comment:
I *think* that `-webkit-overflow-scrolling: touch` is no longer needed on latest iOS, but I could be wrong.

Also `overflow-y: auto` is a bit better than `overflow-y: scroll` because then a scrollbar won’t show if there’s nothing to scroll

Reply:
(just thinking out loud; this was probably left over from before and not introduced in your PR; sorry if so)